calcom/cal.com
# fix: error connecting to pg engine

## What does this PR do?

Fixes #11911 

## Type of change


## How should this be tested?



- stop running instance of cal.com
- delete the existing docker containers <simplest way to do go to do --> docker compose gui tool else docker kill [containerid]
- restart cal.com by ""yarn dx"" 
- should not see this error message any more hopefully 🤞🏼 "@calcom/web:dx: error connecting to engine: sql, type: postgres db: Error: connect ECONNREFUSED 127.0.0.1:5432"

## Mandatory Tasks
